### DISCUSSION TITLE
Firmware and boot issues

### DISCUSSION THREAD

**Discussion by Shruthishettys** [hashicorp/packer-plugin-vsphere#641] [Community Help]
When switching my vSphere’s VM boot mechanism from BIOS to (U)EFI, the virtual cd-rom drive (with windows 11 iso mounted) isn’t detected as a boot device.

ISO File name - [Win_Pro_11_22H2.9_64BIT](url)

Below are the files i see when using the bios.
Also, I attached answer file which i am using for efi
[autounattend.txt](https://github.com/hashicorp/packer-plugin-vsphere/files/14378601/autounattend.txt)

**Comment by tenthirtyam** [comment]
https://github.com/vmware-samples/packer-examples-for-vsphere/discussions/847#discussioncomment-8561307
